src/zkevm_specs/evm/execution/oog_call.py

```diff
@@ -22,55 +15,19 @@ def oog_call(instruction: Instruction):
 
     tx_id = instruction.call_context_lookup(CallContextFieldTag.TxId)
     instruction.call_context_lookup(CallContextFieldTag.IsStatic)
-    # Lookup values from stack
-    instruction.stack_pop()
-    callee_address_rlc = instruction.stack_pop()
-    value = instruction.stack_pop()
-    cd_offset_rlc = instruction.stack_pop()
-    cd_length_rlc = instruction.stack_pop()
-    rd_offset_rlc = instruction.stack_pop()
-    rd_length_rlc = instruction.stack_pop()
-    is_success = instruction.stack_push()
-    instruction.constrain_zero(is_success)
 
-    cd_offset, cd_length = instruction.memory_offset_and_length(cd_offset_rlc, cd_length_rlc)
-    rd_offset, rd_length = instruction.memory_offset_and_length(rd_offset_rlc, rd_length_rlc)
-
-    # Verify memory expansion
-    next_memory_size, memory_expansion_gas_cost = instruction.memory_expansion_dynamic_length(
-        cd_offset,
-        cd_length,
-        rd_offset,
-        rd_length,
-    )
-
-    has_value = 1 - instruction.is_zero(value)
-    callee_address = instruction.rlc_to_fq(callee_address_rlc, N_BYTES_ACCOUNT_ADDRESS)
+    call = CallGadget(instruction, FQ(0), FQ(1), FQ(0), FQ(0))
 
     # TODO: handle PrecompiledContract oog cases
 
-    # Add callee to access list
-    is_warm_access = instruction.read_account_to_access_list(tx_id, callee_address)
+    # Add callee to access list``
```

Comment:
```suggestion
    # Add callee to access list
```

Reply:
fixed in [53df9ee](https://github.com/privacy-scaling-explorations/zkevm-specs/pull/348/commits/53df9ee479bd2b2f70207363595dafb265f91a96)